tools/bouncer_worker/src/queues/modelQueueHandler.js

```diff
@@ -54,7 +56,30 @@ Handler.onMessageReceived = async (cmd, rid, callback) => {
 	};
 
 	try {
-		returnMessage.value = await runBouncerCommand(logDir, cmdParams);
+		const memoryReporting = { enabled: config.elastic.memoryReporting, maxMemory: 0, processTime: 0 };
+		returnMessage.value = await runBouncerCommand(logDir, cmdParams, memoryReporting);
+		if (memoryReporting.maxMemory > 0) {
+			const importFile = cmdParams[3];
```

Comment:
instead of reading the config file again, you should just get decoder to return it to you. 

just add  `file: file: cmdFile.file` in here: https://github.com/3drepo/3drepobouncer/blob/master/tools/bouncer_worker/src/lib/messageDecoder.js#L59